fs-integration-test-suite/accelerator-tests/preconfiguration.steps/src/test/groovy/org/wso2/financial/services/accelerator/test/preconfiguration/steps/CreateConsentAPIAccessRole.groovy

```diff
@@ -0,0 +1,127 @@
+/**
+ * Copyright (c) 2025, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.financial.services.accelerator.test.preconfiguration.steps
+
+import io.restassured.response.Response
+import org.testng.Assert
+import org.testng.annotations.BeforeClass
+import org.testng.annotations.Test
+import org.wso2.financial.services.accelerator.test.framework.FSAPIMConnectorTest
+import org.wso2.financial.services.accelerator.test.framework.constant.ConnectorTestConstants
+import org.wso2.financial.services.accelerator.test.framework.constant.RequestPayloads
+import org.wso2.financial.services.accelerator.test.framework.utility.FSRestAsRequestBuilder
+import org.wso2.financial.services.accelerator.test.framework.utility.TestUtil
+
+import java.nio.charset.Charset
+
+class CreateConsentAPIAccessRole extends FSAPIMConnectorTest {
+
+    String roleManagementUrl
+    String apiResourceUrl
+    String roleId
+    def basicHeader
+    String apiResourceName = "OB-internal-api-resource"
```

Comment:
Shall we define these values as constants

Reply:
Fixed in https://github.com/wso2/financial-services-accelerator/pull/816/commits/72b091ec15dfd639bd78c760097c3f6f402f3f0d